### DISCUSSION TITLE
[Feature Request]: 增加“强制入库“的勾选项

### DISCUSSION THREAD

**Discussion by luowenfu** [jxxghp/MoviePilot#3393] [讨论]
### 当前程序版本

1.5.8

### 运行环境

Docker

### 功能改进类型

主程序

### 功能改进

希望在文件整理时，增加一个”强制入库“的勾选项。
即便是豆瓣、TMDB等，有时候会有找不到媒体信息的冷门资源，希望增加此功能。
![_20240111051101](https://github.com/jxxghp/MoviePilot/assets/62324504/fdf20603-c356-4133-8254-551b7fe859d4)

**Comment by ColaHikari** [comment]
确实觉得该功能很必要
